go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/open-policy-agent/gatekeeper/v3
 
 go 1.22.0
 
+replace github.com/open-policy-agent/frameworks/constraint v0.0.0-20240927180816-0f64229c5539 => github.com/abhipatnala/frameworks/constraint v0.0.0-20241010172729-61e15952c5c5
```

Comment:
@abhipatnala can you pls open a PR on framework as well so we can review and get it merged before merging this one? thanks!

Reply:
@ritazh  Here is the Pr for removing cel driver from framework: https://github.com/open-policy-agent/frameworks/pull/486

Reply:
@abhipatnala PTAL and remove the go.mod override 😄